### DISCUSSION TITLE
MenuItem as link

### DISCUSSION THREAD

**Discussion by IHIutch** [adobe/react-spectrum#4958] [Q&A]
Is there a way to make a MenuItem a Link, whether using the link component or `a` element? In my case, I'm specifically referring to React Aria Component implementation

- Related: https://github.com/adobe/react-spectrum/discussions/3620
- Also related, but unresolved https://github.com/adobe/react-spectrum/issues/1244

**Comment by snowystinger** [comment]
Not currently. Right now, navigation by selection must be done programmatically. We are in the middle of discussing how we would implement this, and we hope to have a general approach to try soon.
